rfcs/text/0021/threat.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: threat
+
+  fields:
+
+  - name: enrichments
+    level: extended
+    type: nested
+    short: List of indicators enriching the event
+    description: >
+      A list of associated indicators enriching the event, and the context of that association/enrichment
+
+  - name: enrichments.matched.atomic
```

Comment:
@ebeahan I need to add the rest of the `matched` fields from above here, but I first wanted to verify how those should be represented given the `nested` questions above.

Reply:
Sorry for the delay in responding here @rylnd!

For a `nested` field, you'll need to define the `nested` field itself as `type: nested` and then each individual field underneath that `nested` object:

```yaml
    - name: enrichments.matched
      description: >
        Section information of the ELF file.
      type: nested
      level: extended

  - name: enrichments.matched.atomic
     level: extended
     short: Indicator atomic match
     ...
```

Reply:
`matched` is not intended to be `nested`, only `enrichments` is. So it sounds like the current format is correct, then?

Reply:
Sorry, I misunderstood. You're correct; the format looks good to me. 👍